model-server/src/test/kotlin/org/modelix/model/server/ModelClientV2Test.kt

```diff
@@ -154,4 +155,30 @@ class ModelClientV2Test {
 
         assertEquals("localhost", modelClient.getUserId())
     }
+
+    @Test
+    fun `newly created repository can be removed`() = runTest {
+        val url = "http://localhost/v2"
+        val client = ModelClientV2.builder().url(url).client(client).build().also { it.init() }
+        val repositoryId = RepositoryId(UUID.randomUUID().toString())
+        client.initRepository(repositoryId)
+
+        val success = client.deleteRepository(repositoryId)
+
+        assertEquals(true, success)
+    }
+
+    @Test
+    fun `non-existing repository cannot be removed`() = runTest {
+        val url = "http://localhost/v2"
+        val client = ModelClientV2.builder().url(url).client(client).build().also { it.init() }
+        val repositoryId = RepositoryId(UUID.randomUUID().toString())
+        client.initRepository(repositoryId)
+
+        client.deleteRepository(repositoryId)
+        // repository should not exist here
```

Comment:
Can't this be dropped. If a truly random repo name was never created as a repo it should most likely not exist.

Reply:
fixed in [c212413](https://github.com/modelix/modelix.core/pull/450/commits/c212413f2b2c28a560af40327abf88e0abdd42e0)